scripts/import-ai-lab-samples

```diff
@@ -4,16 +4,22 @@ ROOT_DIR=$(realpath $SCRIPTDIR/..)
 SKELETON_DIR=$ROOT_DIR/skeleton
 TEMPLATE_DIR=$ROOT_DIR/templates
 
-REPO="https://github.com/redhat-ai-dev/ai-lab-samples"
+REPO="${1:-https://github.com/redhat-ai-dev/ai-lab-samples}"
```

Comment:
I think it might be best to use env vars instead of arguments here. Just off the top of my head if I only wanted to change the branch I have to also provide the 1st argument even if its the default, no?

Reply:
updated